lib/members.ex

```diff
@@ -0,0 +1,62 @@
+defmodule GlassFactoryApi.Members do
+  @moduledoc """
+  Module which hold the GET functions of organization members.
+  """
+  alias GlassFactoryApi.ApiClient
+  alias GlassFactoryApi.Members.Member
+
+  @doc """
+  List the members of a organization
+
+  ## Examples
+
+      iex> GlassFactoryApi.Member.list()
+      [ %Member{
+          name: "John Doe",
+          email: "john.doe@example.org",
+          id: 0,
+          archived: false,
+          capacity: 8.0,
+          freelancer: false,
+          joined_at: "2019-01-01"
+        }
+      ]
+  """
+  @spec list_members() :: list(Member.t())
+  def list_members() do
+    {:ok, %{status_code: 200, body: body}} = ApiClient.get("members")
```

Comment:
If I use something like `ApiClient.get("xablau")` what will be the response? 🤔 

Not sure about that, but do you think it's a good idea to do something like that in the `lib/api_client.ex`:

```ex
defp url("members"), do: "#{api_domain()}/api/public/v1/members"
defp url(_), do: raise("The given URL is not available")
```

or

```ex
@allowed_urls ["members"]

defp url(resource) do 
  if resource in @allowed_urls do
    "#{api_domain()}/api/public/v1/#{resource}"
  else
    raise("The given URL is not available")
  end
end
```
?

Reply:
Nowadays the `ApiClient` will respond with something like:

```elixir
{:error, %{status_code: 404... }}
``` 

The whole idea of the `ApiClient` is just to normalize all requests as an internal client and the methods can pass the path they are requesting. 

Doing as you said will break requests like `"members/#{member_id}" or I need to add more complexity to handle those cases. 

I think for now it's worth to let the module decide which URL it'll call and let the API returns with `404` if it's the case, WDYT?